README.md

```diff
@@ -381,7 +383,8 @@ const factory = Model.UserFactory.getInstance({
 });
 const config = {
     params: {
-        username: 'batman'
+        username: 'batman',
+        scmContext: 'github:github.com'
```

Comment:
It's just a unique name generated by scm-module with free format. `scm-github` generates "github:hostname" and`scm-bitbucket` generates "bitbuket:hostname". Another custom plugin like `scm-assembla` may generate "assembla" because of a single hosted service.